app/components/image-slider-component.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  ajax: Ember.inject.service('restaurants-service'),
+
+  init() {
+    this._super.apply(this,arguments);
+
+    this.get('ajax').getRestaurant(1).done(data => {
```

Comment:
There should be getRestaurants service call where you can fetch multiple restaurants.